doc/api/dgram.md

```diff
@@ -396,6 +396,26 @@ decremented to 0 by a router, it will not be forwarded.
 The argument passed to to `socket.setMulticastTTL()` is a number of hops
 between 0 and 255. The default on most systems is `1` but can vary.
 
+### socket.setRecvBufferSize(size)
+<!-- YAML
```

Comment:
Would it make sense to also include these as options when the `Socket` is created?